test/junit/scala/util/PropertiesTest.scala

```diff
@@ -37,7 +37,8 @@ class PropertiesTest {
     assertEquals(Properties.envOrElse("_PropertiesTest_NOT_DEFINED", "test"), "test")
 
     var done = false
-    assertNotEquals(Properties.envOrElse("JAVA_HOME", { done = true; "bar" }), "bar")
+    val envName = System.getenv().keySet().iterator().next()
+    assertNotEquals(Properties.envOrElse(envName, { done = true; "bar" }), "bar")
```

Comment:
I'm so ignorant, I don't about environments where the map is empty. You could harden it here, or kick that down the road.